internal/controller/policyserver_controller_deployment.go

```diff
@@ -297,6 +297,10 @@ func configureLabelsAndAnnotations(policyServerDeployment *appsv1.Deployment, po
 	}
 	policyServerDeployment.Labels[constants.AppLabelKey] = policyServer.AppLabel()
 	policyServerDeployment.Labels[constants.PolicyServerLabelKey] = policyServer.Name
```

Comment:
We should add a card to schedule removing the old labels in the future.

Reply:
Even better, we have a dedicated PR: https://github.com/kubewarden/kubewarden-controller/pull/1038

This is currently under the blocked column and is assigned to Kubewarden 1.25